skimage/morphology/_flood_fill.py

```diff
@@ -46,6 +46,11 @@ def flood_fill(image, seed_point, new_value, *, selem=None, connectivity=None,
         If a tolerance is provided, adjacent points with values within plus or
         minus tolerance from the seed point are filled (inclusive).
     inplace : bool, optional
```

Comment:
Nit: Can we change the order of these two parameters. Have the suggestion one appear first?